### PR TITLE
[WFLY-16137] - update elytron docs for WFCORE-4314

### PR DESCRIPTION
**Pull request by baranowb**
Issue: https://issues.redhat.com/browse/WFLY-16137
Related: https://issues.redhat.com/browse/WFCORE-4314

### CODE REVIEW

File: docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc [hunk 1]
```diff
@@ -1176,6 +1176,185 @@ generate an example key pair:
 /subsystem=elytron/key-store=httpsKS:store()
 ----
 
+[[list-all-aliases-recursively-in-a-key-store.]]
+==== List all aliases(recursively) in a key store.
```
Comment (fjuma) [inline docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc:1180]: Minor nit but would be good to have a space before (recursively).

File: docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc [hunk 2]
```diff
@@ -1176,6 +1176,185 @@ generate an example key pair:
...
+                "version" => "v3"
+            }]
+        },
+        "localhost2" => {...}
+    }
+}
+----
+
+[[list-all-aliases-recursively-no-keys-in-a-key-store.]]
+==== List all aliases(recursively, no keys) in a key store.
```
Comment (fjuma) [inline docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc:1239]: Again, would be good to have a space before the opening bracket.

Comment (fjuma) [reply docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc:1239]: Might be good to reword this to (recursively, non-verbose)

File: docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc [hunk 3]
```diff
@@ -1176,6 +1176,185 @@ generate an example key pair:
...
+                "version" => "v3"
+            }]
+        },
+        "localhost2" => {...}
+    }
+}
+----
+
+[[read-alias-in-a-key-store.]]
+==== Read alias in a key store.
```
Comment (fjuma) [inline docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc:1273]: s/Read alias/Read an alias

File: docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc [hunk 4]
```diff
@@ -1176,6 +1176,185 @@ generate an example key pair:
...
+
+[source,options="nowrap"]
+----
+/subsystem=elytron/credential-store=exampleCS:read-alias(name="localhost")
```
Comment (fjuma) [inline docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc:1277]: s/credential-store=exampleCS/key-store=httpsKS

File: docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc [hunk 5]
```diff
@@ -1176,6 +1176,185 @@ generate an example key pair:
...
+                "signature-algorithm" => "SHA256withDSA",
+                "signature" => "30:3c:02:1c:4f:1a:e9:ea:c2:0a:23:21:91:e8:aa:58:ce:df:b8:a1:ff:02:90:cb:33:b1:99:b2:ca:16:f6:5a:02:1c:38:2a:19:17:fa:c3:a1:1a:dd:bb:fe:96:e9:3a:6d:fa:e9:a6:63:4c:9f:fb:db:ec:dc:49:1e:35",
+                "version" => "v3"
+        }]
+    }
+}
+----
+
+[[read-alias-in-a-key-store-non-verbose.]]
+==== Read alias in a key store(non verbose).
```
Comment (fjuma) [inline docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc:1329]: s/Read alias in a key store/Read an alias in a key store (non-verbose)

File: docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc [hunk 6]
```diff
@@ -1176,6 +1176,185 @@ generate an example key pair:
...
+
+[source,options="nowrap"]
+----
+/subsystem=elytron/credential-store=exampleCS:read-alias(name="localhost",verbose=false)
```
Comment (fjuma) [inline docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc:1333]: s/credential-store=exampleCS/key-store=httpsKS

File: docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc [hunk 7]
```diff
@@ -1176,6 +1176,185 @@ generate an example key pair:
...
+    }
+}
+----
+
+[[list-all-aliases-recursively-no-keys-in-a-key-store.]]
+==== List all aliases(recursively, no keys) in a key store.
+
+[source,options="nowrap"]
+----
+/subsystem=elytron/credential-store=exampleCS:read-aliases(recursive=true, verbose=false)
```
Comment (fjuma) [inline docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc:1243]: s/credential-store=exampleCS/key-store=httpsKS

**Review by fjuma — CHANGES_REQUESTED** [review]
Thanks for the PR @baranowb!

I've added some small comments.

In the commit message and PR title/description, please reference this WFLY issue instead of the WFCORE issue:

https://issues.redhat.com/browse/WFLY-16137

Thanks!

**Review by fjuma — APPROVED** [review]
Thanks @baranowb!

**Comment by bstansberry** [comment]
Thanks, @baranowb 

I went ahead and merged this before the 19.0.0.Beta6 upgrade, as I know this will be in it and I wanted to merge the 26.x variant for this week's 26.1 beta.
